discussion/app/discussion/api/DiscussionApi.scala

```diff
@@ -182,7 +183,10 @@ trait DiscussionApiLike extends Http with ExecutionContexts with Logging {
   }
 
   override protected def getJsonOrError(url: String, onError: (WSResponse) => String, headers: (String, String)*) = {
-    super.getJsonOrError(url, onError, headers :+ guClientHeader: _*)
+    if(DiscussionStopTrafficFromDotcom.isSwitchedOn)
+      throw ServiceUnavailableException("Request from Dotcom to Discussion are currently unavailable")
```

Comment:
The return type of this method is a `Future` I think. It would be better to return a Future.failed(...) rather than throwing an exception